.azure-pipelines/build-template.yml

```diff
@@ -39,7 +39,7 @@ jobs:
   timeoutInMinutes: ${{ parameters.timeout }}
 
   container:
-    image: sonicdev-microsoft.azurecr.io:443/${{ parameters.sonic_slave }}:latest
+    image: sonicdev-microsoft.azurecr.io:443/${{ parameters.sonic_slave }}:bookworm
```

Comment:
Automation pipeline didn't upload sonic-slave-bookworm:bookworm image.
Is it uploaded by you manually?